configure.ac

```diff
@@ -96,10 +96,16 @@ ARG_ENABL_SET([kex-ntru], [enable KEX-NTRU.])
 AM_CONDITIONAL([ntru], [test "x$kex_ntru" = xtrue])
 AM_CONDITIONAL([USE_NTRU], [test "x$kex_ntru" = xtrue])
 
+ARG_ENABL_SET([picnic], [enable PICNIC.])
+AM_CONDITIONAL([picnic], [test "x$picnic" = xtrue])
+AM_CONDITIONAL([USE_PICNIC], [test "x$picnic" = xtrue])
+
 ARG_ENABL_SET([kex-sidh-iqc-ref], [enable KEX-SIDH-IQC-REF.])
 AM_CONDITIONAL([kex_sidh_iqc_ref], [test "x$kex_sidh_iqc_ref" = xtrue])
 AM_CONDITIONAL([USE_SIDH_IQC], [test "x$kex_sidh_iqc_ref" = xtrue])
 
+
+AM_CPPFLAGS="-g -std=gnu11 -Wno-unused-function -Werror -Wpedantic -Wall -Wextra -DCONSTANT_TIME"
```

Comment:
It's a duplicate, there is another AM_CPPFLAGS below. Needs to be removed. Merge artifact.